kouncil-backend/src/main/java/com/consdata/kouncil/security/function/FunctionService.java

```diff
@@ -0,0 +1,21 @@
+package com.consdata.kouncil.security.function;
+
+import com.consdata.kouncil.model.admin.SystemFunctionName;
+import com.consdata.kouncil.security.function.dto.SystemFunctionDto;
+import java.util.List;
+import java.util.stream.StreamSupport;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+
+@Service
+@RequiredArgsConstructor
+public class FunctionService {
```

Comment:
SystemFunctionService? Also SystemFunctionController and SystemFunctionConverter